Makefile

```diff
@@ -13,7 +13,7 @@ format:
 
 .PHONY: test
 test:
-	python3 -m tox -p auto
+	python3 -m tox -p auto -o
```

Comment:
The output gets very messy when run in parallel. I think we should turn off the `-p` option if we deem readable test output more valuable than those few extra seconds it takes to run (I do).

Reply:
Removed parallel testing.

Reply:
`tox` still runs tests in parallel, the `-o` flag just ensures that the output is more suitable to a CI/CD environment (e.g. no spinner): https://tox.readthedocs.io/en/latest/example/basic.html#parallel-mode

Reply:
Ah, I see that was the old commit, never mind 😅